contracts/payloads/IGP30/PayloadIGP30.sol

```diff
@@ -0,0 +1,759 @@
+        vault_ = VAULT_T1_FACTORY.deployVault(
+            address(VAULT_T1_DEPLOYMENT_LOGIC),
+            abi.encodeWithSelector(
+                IFluidVaultT1DeploymentLogic.vaultT1.selector,
+                vaultConfig.supplyToken,
+                vaultConfig.borrowToken 
+            )
+        );
+
+        // Set user supply config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserSupplyConfig[]
+                memory configs_ = new AdminModuleStructs.UserSupplyConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserSupplyConfig({
+                user: address(vault_),
+                token: vaultConfig.supplyToken,
+                mode: vaultConfig.supplyMode,
+                expandPercent: vaultConfig.supplyExpandPercent,
+                expandDuration: vaultConfig.supplyExpandDuration,
+                baseWithdrawalLimit: getRawAmount(
+                    vaultConfig.supplyToken,
+                    vaultConfig.supplyBaseLimitInUSD,
+                    true
+                )
+            });
+
+            LIQUIDITY.updateUserSupplyConfigs(configs_);
+        }
+
+        // Set user borrow config for the vault on Liquidity Layer.
+        {
+            AdminModuleStructs.UserBorrowConfig[]
+                memory configs_ = new AdminModuleStructs.UserBorrowConfig[](1);
+
+            configs_[0] = AdminModuleStructs.UserBorrowConfig({
+                user: address(vault_),
+                token: vaultConfig.borrowToken,
+                mode: vaultConfig.borrowMode,
+                expandPercent: vaultConfig.supplyExpandPercent,
+                expandDuration: vaultConfig.borrowExpandDuration,
+                baseDebtCeiling: getRawAmount(
+                    vaultConfig.borrowToken,
+                    vaultConfig.borrowBaseLimitInUSD,
+                    false
+                ),
+                maxDebtCeiling: getRawAmount(
+                    vaultConfig.borrowToken,
+                    vaultConfig.borrowMaxLimitInUSD,
+                    false
+                )
+            });
+
+            LIQUIDITY.updateUserBorrowConfigs(configs_);
+        }
+
+        // Update core settings on vault.
+        {
+            IFluidVaultT1(vault_).updateCoreSettings(
+                vaultConfig.supplyRateMagnifier,
+                vaultConfig.borrowRateMagnifier,
+                vaultConfig.collateralFactor,
+                vaultConfig.liquidationThreshold,
+                vaultConfig.liquidationMaxLimit,
+                vaultConfig.withdrawGap,
+                vaultConfig.liquidationPenalty,
+                vaultConfig.borrowFee
+            );
+        }
+
+        // Update oracle on vault.
+        {
+            IFluidVaultT1(vault_).updateOracle(vaultConfig.oracle);
+        }
+
+        // Update rebalancer on vault.
+        {
+            IFluidVaultT1(vault_).updateRebalancer(
+                0x264786EF916af64a1DB19F513F24a3681734ce92
```

Comment:
```suggestion
                address(FLUID_RESERVE)
```

same thing but clearer because we also use this for copy paste in the future likely